networking/aws-load-balancer.adoc

```diff
@@ -0,0 +1,14 @@
+:_content-type: ASSEMBLY
+[id="aws-load-balancer"]
+= AWS Load Balancer Operator
+include::_attributes/common-attributes.adoc[]
+:context: aws-load-balancer
+
+toc::[]
+
+
+The AWS Load Balancer (ALB) Operator  is a Red-Hat supported Operator that users can optionally install on SRE-managed {product-title} clusters.  The AWS Load Balancer (ALB) Operator manages the life cycle of the AWS-managed AWS Load Balancer Controller (ALBC) that provisions Amazon Elastic Load Balancing v2 services for applications running in {product-title} clusters.
```

Comment:
I think AWS recommended using just Elastic Load Balancing (ELB) or AWS Elastic Load Balancing where ambiguous. Please refer to the AWS Docs meeting on July 12.

Reply:
> I think AWS recommended using just Elastic Load Balancing (ELB) or AWS Elastic Load Balancing where ambiguous. Please refer to the AWS Docs meeting on July 12.

Ack. Changed.

Reply:
> > I think AWS recommended using just Elastic Load Balancing (ELB) or AWS Elastic Load Balancing where ambiguous. Please refer to the AWS Docs meeting on July 12.
> 
> Ack. Changed.

![image](https://github.com/openshift/openshift-docs/assets/122639474/c3ad7c32-880f-483a-8c07-1867017b91a4)

Reply:
![image](https://github.com/openshift/openshift-docs/assets/122639474/00cb4f97-c42a-488d-bee9-417644a0f015)